cache.py

```diff
@@ -1,11 +1,22 @@
 import os
 from collections import OrderedDict
 import uuid
+from dataclasses import dataclass
 
 from pytube import YouTube
 
 from urllib.parse import urlparse, parse_qs
 
+@dataclass
+class VideoInfo():
```

Comment:
add a field in here called `size_bytes` which will be an integer representing how big the file is on disk

we can then use this field in `_downsize_cache_to_target_bytes` instead of calculating it with `os`

Reply:
done